js/scripts.js

```diff
@@ -0,0 +1,12 @@
+var a = prompt('Wstaw wartość a:');
+	b = prompt('Wstaw wartość b:');
+	value = (a * a) - (2 * a * b) + (b * b);
+console.log('Wynik to: ' + value)
+
+if (value > 0) {
+    alert('Wynik dodatni');
+} else if (value < 0) {
+    alert('Wynik ujemny');
+} else {
+    alert('Wynik równy zero!');
```

Comment:
are you sure that in this case result will be equal to 0? what result you get, when user type a letter in prompt?